src/assets/content/dictionary/aesthetic.md

```diff
@@ -1,6 +1,10 @@
 ---
 title: Aesthetic
-definition:
-sources: []
-perspectives: []
+definition:"Aesthic is a particular conception of beauty or art, a particular taste for or approach to what is pleasing to the senses and especially sight"
```

Comment:
```suggestion
definition: Aesthic is a particular conception of beauty or art, a particular taste for or approach to what is pleasing to the senses and especially sight.
```
Bug: I think the lack of a space between `definition:` and `Aesthetic` is breaking the build. You can accept this suggestion via the Github UI, or you can make an additional commit to this PR, as you like.